src/styles/_sizes.media.scss

```diff
@@ -0,0 +1,48 @@
+$small-width: 480px;
```

Comment:
I would probably move this along with where its being used, since it will likely be used in the grid only and not globally